test/unit/service/JobServiceTests.ts

```diff
@@ -77,23 +77,52 @@ describe('JobService', function () {
         expect(error).to.equal('Could not create job');
       });
 
-      
-      it('should throw an error when axios returns a negative id number', async () => {
-        const url: string= process.env.API_URL + 'add-job';
-        const mock = new MockAdapter(axios);
-        mock.onPost(url).reply(500);
-
-        let error;
-        try {
-          await jobService.addJob(jobRequest);
-        } catch (e) {
-          error = e.message;
-        }
-
-        expect(error).to.equal('Could not create job');
-
-      });
-
     });
   });
+
+  it('should return a validation response when job data is invalid', async () => {
```

Comment:
it looks like you're not mocking out the validator for testing the service. mock it out and get it to return valid and invalid scenario and have a unit test to make sure it handles each